lib/search.rb

```diff
@@ -92,4 +70,18 @@ def records_to_tsv
       end
     end
   end
+
+  private
+
+  # Only used by num_results
+  def search_uri
```

Comment:
It's a little bit annoying that we have to duplicate the query construction logic from `solr_cursorstream`; I think if `default_params` https://github.com/mlibrary/solr_cursorstream/blob/main/lib/solr/cursorstream.rb#L98 were public that would help, though?

Reply:
OK. In the meantime I'll go ahead and merge this; I'll make a note on DEV-285.

Reply:
See https://hathitrust.atlassian.net/browse/DEV-356